src/patterns/exit-a-page-quickly/safety-content-example/index.njk

```diff
@@ -0,0 +1,54 @@
+---
+title: Example of safety content page
+layout: layout-example-full-page.njk
+---
+
+{% extends "example-wrappers/full-page.njk" %}
+
+{% block content %}
+		<h1 class="govuk-heading-l">Ways to stay safe online</h1>
+
+		<p class="govuk-body"> It can be easy for someone to see what you’re doing online, especially if you’re using a home computer.</p>
+
+		<p class="govuk-body">You can make it more difficult for them by taking some precautions.</p>
+
+		<p class="govuk-body">If you think your devices or internet search activities are being monitored, try to use a device that is not being monitored. That should be a device that the person does not or has not had physical or remote access to. This is the safest thing to do if you do not want someone to know that you’re visiting certain websites.</p>
+
+		<h2 class="govuk-heading-m">Cover your tracks online</h2>
+
+		<p class="govuk-body">Every time you use the internet, your browser stores a record of where you’ve been online with:</p>
+		<ul class="govuk-list govuk-list--bullet">
+			<li>a list of the pages you’ve looked at and the files you’ve downloaded</li>
+			<li>small files called cookies which remember your settings for different sites</li>
+		</ul>
+
+		<h2 class="govuk-heading-m">Deleting browser history</h2>
+
+		<p class="govuk-body">You can improve your safety by deleting your browser history. But you need to be careful because:</p>
+		<ul class="govuk-list govuk-list--bullet">
+			<li>deleting cookies will also get rid of stored passwords for online accounts</li>
+			<li>clearing your history could make someone more suspicious</li>
+		</ul>
```

Comment:
```suggestion
		<h2 class="govuk-heading-m">Deleting internet browsing history</h2>

		<p class="govuk-body">You can improve your safety by deleting your internet browsing history. But you need to be careful because:</p>
		<ul class="govuk-list govuk-list--bullet">
			<li>deleting cookies will also get rid of stored passwords for online accounts</li>
			<li>clearing your history could make someone more suspicious</li>
		</ul>
```